pkg/monitor/agent/listener1_2.go

```diff
@@ -78,3 +81,11 @@ func (ml *listenerv1_2) drainQueue() {
 func (ml *listenerv1_2) Version() listener.Version {
 	return listener.Version1_2
 }
+
+// Close closes the underlying socket and .
```

Comment:
And what? Don't leave me hanging here michi

Reply:
got distracted while writing this comment :) updating...